main.py

```
@@ -103,90 +115,108 @@ async def _handle_network_data(data):
     finally:
         await network.stop()
 
 
+# -------------------------
+# Run Node
+# -------------------------
 async def _run_node(network, chain, mempool, pending_nonce_map, get_next_nonce):
     await network.start()
 
     alice_sk, alice_pk = create_wallet()
     bob_sk, bob_pk = create_wallet()
```

Ruff (0.15.2)

[warning] Unpacked variable `bob_sk` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)